schedule/yast/maintenance/lvm_thin_provisioning_sle15sp3_dev.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: lvm_thin_provisioning_dev
+vars:
+  DUD_ADDONS: sdk
+  YUI_REST_API: 1
+schedule:
+  suggested_partitioning:
+    - installation/partitioning/new_partitioning_gpt
+  system_role:
+    - installation/system_role/accept_selected_role_SLES_with_GNOME
```

Comment:
keys should be in order even if it is not necessary, anyway, gnome is better as default, not need to overwrite.